sections/collection-list.liquid

```diff
@@ -31,7 +31,7 @@
     {% endunless %}
   
     <slider-component class="slider-mobile-gutter">
-      <ul class="collection-list contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %} grid grid--1-col{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %} grid--{{ section.blocks.size }}-col-tablet{% if section.settings.swipe_on_mobile %} slider slider--tablet grid--peek{% endif %} collection-list--{{ section.blocks.size }}-items"
+      <ul class="collection-list contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %} grid{% if section.settings.columns_desktop == 1 %} grid--full-width{% else %} grid--{{ section.settings.columns_desktop }}-col-desktop{% endif %} grid--{{ section.settings.columns_mobile }}-col-tablet-down{% if section.settings.swipe_on_mobile %} slider slider--tablet grid--peek{% endif %} collection-list--{{ section.blocks.size }}-items"
```

Comment:
Also it looks like we're not hiding the slider controls properly when there are 2 blocks 2 columns and swipe on mobile: 
![](https://screenshot.click/18-22-95xmn-v25cl.png)

Reply:
Good find! Here is the update: https://github.com/Shopify/dawn/pull/1352/commits/280f6eaf83c3caa869ceb70affc5f928b71e1d42

First I had to convert the number of columns on mobile to an integer (couldnt compare it since it was a string). Then I compared the numbers. If it was the same it meant we didn't need it. 1 column == 1 block or 2 columns == 2 blocks